frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -481,8 +482,14 @@ const CreateBackingStoreForm: React.FC<CreateBackingStoreFormProps> = withHandle
     providerDataReducer,
     initialState,
   );
+  const [disabled, setDisabled] = React.useState(true);
 
-  const { cancel, close, inProgress, errorMessage, handlePromise, isPage } = props;
+  React.useEffect(() => {
+    if (bsName.length > 0 && providerDataState.target.length > 0) setDisabled(false);
+    else setDisabled(true);
+  }, [bsName, providerDataState]);
```

Comment:
What's the use case for setting it through state, why not using `isRequired` ?
The button itself wont submit without it and will display a message.
This would reduce extra costs on UI end too.

Reply:
Also, are we allowing user to create backing store without selecting secret ?
I think secret should be selected as required too.

Reply:
There are scenarios when secret is not required. Hence no need to enforce it.

Reply:
`isRequired` does not disable the button. It adds the *. But the request still goes through.

Reply:
yeah! and you will get error message to fill out the field.
I feel it should be like that and this is how it is at various places in openshift.